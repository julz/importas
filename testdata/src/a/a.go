package a

import (
	wrong_alias "fmt"      // want `import "fmt" imported as "wrong_alias" but must be "fff" according to config`
	wrong_alias_again "os" // want `import "os" imported as "wrong_alias_again" but must be "stdos" according to config`
)

func ImportAsWrongAlias() {
	wrong_alias.Println("foo")
	wrong_alias_again.Stdout.WriteString("bar")
}
