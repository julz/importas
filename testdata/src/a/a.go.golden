package a

import (
	fff "fmt" // want `import "fmt" imported as "wrong_alias" but must be "fff" according to config`
	"io"
	stdos "os" // want `import "os" imported as "wrong_alias_again" but must be "stdos" according to config`
)

func ImportAsWrongAlias() {
	fff.Println("foo")
	stdos.Stdout.WriteString("bar")
	io.Pipe()
}
